telehealth-example/src/main/java/com/pubnub/components/example/getting_started/ui/view/Chat.kt

```diff
@@ -0,0 +1,203 @@
+import kotlinx.coroutines.flow.Flow
+
+object Chat {
+
+    @Composable
+    internal fun Content(
+        messages: Flow<PagingData<MessageUi>>,
+        presence: Presence? = null,
+        onMessageSelected: (MessageUi.Data) -> Unit,
+        onReactionSelected: ((React) -> Unit)? = null,
+        patientUUID: String,
+        patientName: String
+    ) {
+
+        val customTheme = ThemeDefaults.messageList(
+            messageOwn = ThemeDefaults.message(
+                text = text(
+                    Modifier
+                        .background(color = MessageOwnBackgroundColor)
+                        .padding(2.dp)
+
+                ),
+                shape = ShapeThemeDefaults.shape(shape = Shapes.medium),
+                modifier = Modifier
+            ),
+            message = ThemeDefaults.message(
+                text = text(
+                    modifier = Modifier
+                        .background(color = MessageBackgroundColor)
+                        .padding(2.dp)
+                ),
+                shape = ShapeThemeDefaults.shape(shape = Shapes.medium),
+                modifier = Modifier.clip(RoundedCornerShape(10.dp))
+            )
+        )
+        val localFocusManager = LocalFocusManager.current
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .pointerInput(Unit) {
+                    detectTapGestures(onTap = {
+                        localFocusManager.clearFocus()
+                    })
+                }
+        ) {
+            MessageListTheme(customTheme) {
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .background(color = Color.hsl(196F, 0.65F, 0.57F))
+                ) {
+                    Image(
+                        modifier = Modifier
+                            .size(36.dp)
+                            .padding(top = 16.dp),
+                        painter = painterResource(id = R.drawable.chevron),
+                        contentDescription = "logo"
+                    )
+                    Spacer(modifier = Modifier.width(80.dp))
+                    Column {
+                        Text(
+                            text = patientName,
+                            modifier = Modifier.padding(top = 8.dp, start = 32.dp),
+                            color = Color.White,
+                            fontSize = 16.sp,
+                            fontWeight = FontWeight.Bold
+                        )
+                        Text(
+                            text = patientUUID,
+                            modifier = Modifier.padding(start = 20.dp, bottom = 10.dp),
+                            color = Color.White,
+                            fontSize = 16.sp,
+                            fontWeight = FontWeight.Bold
+                        )
+                    }
+                }
+                MessageList(
```

Comment:
Message list item is not filling the whole space, the ripple looks weird.
<img width="432" alt="image" src="https://user-images.githubusercontent.com/7523141/197480592-c007eb07-3d4a-49a2-8f86-424f1a7325e9.png">

Reply:
Done